### ISSUE TITLE
Add a page showcasing experince of past GSoCers

### ISSUE THREAD

**Issue by ZER-0-NE** [NIT-dgp/nit-dgp.github.io#40]
I see many past GSoCers @ananyo2012 @PaliwalSparsh @pandusonu2 @Aniq55  were gracious enough to share their experience on the [facebook page](https://www.facebook.com/groups/NITDgpOS/).
I think it would be great to have a page sharing all these experiences all stacked up in one place.

**Comment by ananyo2012** [comment]
Yeah I think it is a good idea to start archiving experiences so that we
can share the link instead of reposting everything
